programs/leap-util/actions/chain.cpp

```diff
@@ -47,6 +47,16 @@ void chain_actions::setup(CLI::App& app) {
       // properly return err code in main
       if(rc) throw(CLI::RuntimeError(rc));
    });
+
+  auto* sstate =  sub->add_subcommand("last-shutdown-state", "indicate whether last shutdown was clean or not");
+  sstate->add_option("--state-dir,-o", opt->sstate_state_dir, "The location of the state directory (absolute path or relative to the current directory)")->capture_default_str();
```

Comment:
Drop the `-o` short option name and move this option to the parent command `chain-state`.

Reply:
Thank you, done!